src/components/home/Matches.js

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
+import { connect } from 'react-redux';
+import { array, func } from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import { getMatches } from 'actions/matchesActions';
+import UserAvatar from 'resources/icons/UserAvatar';
 
-const Matches = () => (
-  <h2 className="subtitle center">
-    <FormattedMessage id="home.sidebar.noMatches" />
-  </h2>
-);
+class Matches extends Component {
+  componentDidMount() {
+    const { matches, getMatches } = this.props;
 
-export default Matches;
+    !matches && getMatches();
+  }
+
+  render() {
+    const { matches } = this.props;
+
+    return (
+      <Fragment>
+        {matches && matches.length ?
+          (
+            <h2 className="subtitle center">
+              <FormattedMessage id="matches.list.title" />
+              <div className="list">
+                {matches.map((match, key) => (
+                  <div className="list-item" key={key}>
+                    <div className="smallthumb">
+                      <span className="avatar" onClick={this.toggleSideMenu}>
+                        {match.user.avatar.smallThumbUrl ? <div className="preview"><img alt="Avatar" src={match.user.avatar.smallThumbUrl} /></div> : <UserAvatar />}
```

Comment:
Can you split this line?

Anyway, I commented on your previous PR about refactoring part of this into a new component.